### ISSUE TITLE
create controller programmatically throws exception

### ISSUE THREAD

**Issue by thisiscam** [bfeher/BFPaperTabBarController#3]
I'm experimenting with this controller and figured that it raises NSRange out of bound exception when calling alloc init. it seems that alloc init somehow calls initwithnib, which I don't have any. Is this control only supposed to be used with storyboard?

**Comment by Timstarockz** [comment]
If it's not too late, I've modified this class to work programmatically! You can check out the source here! I basically removed all references to IB and refactored/reorganized some of the initialization methods. Now it works like a charm without IB!

https://github.com/Timstarockz/BFPaperTabBarController/tree/master/Classes

**Comment by benvium** [comment]
@Timstarockz Thank you!

**Comment by bfeher** [comment]
@Timstarockz Thanks! I'll take a closer look next week and perhaps make this the standard.

**Comment by tobatha** [comment]
@bfeher so rite now with the latest version, init tab bar controller programmatically is possible?

**Comment by bfeher** [comment]
@tobatha It hasn't been integrated into the main branch, so please use Timstarockz version above. Thank you.

**Comment by tobatha** [comment]
@bfeher yeah, i tried last nite, seems no luck, but won't crash anymore, that's very reassuring! ^_^

**Comment by bfeher** [comment]
@tobatha I'm sorry that I have been criminally negligent of my repos lately. Life is getting too busy. If you find anything that needs fixed please let me know and I will try to fix it ASAP :)
